Knossos.NET/Views/GlobalSettingsView.axaml

```diff
@@ -314,6 +316,7 @@
 
 				<!--INPUT SETTINGS-->
 				<Label Margin="5,15,0,0" FontWeight="Bold" FontSize="18" ToolTip.Tip="Input settings where you can specify which of your plugged-in devices FSO can use in-game.">Input</Label>
+				<TextBox BorderBrush="Transparent" IsReadOnly="True" TextWrapping="Wrap" Margin="5,0,5,0" Foreground="Yellow">As of FSO version 24.2 input settings not longer have any effect and is only intended to use with old versions. On 24.2 or newer you can change settings ingame by going to "Options" and pressing F3 to display all settings.</TextBox>
```

Comment:
`Note, input settings for most mods using FSO version 24.2.0 or higher will need to be changed in-game by default. You can change in-game settings by going to "Options" menu of the mod you are playing.`